src/app/static/src/app/components/plots/useUpdateScale.ts

```diff
@@ -0,0 +1,22 @@
+import {useStore} from "vuex";
+import {RootState} from "../../root";
+import {ScaleSettings} from "../../store/plotState/plotState";
+import {computed} from "vue";
+
+export const useUpdateScale = () => {
+    const store = useStore<RootState>();
+    const selectedIndicator = computed(() => {
+        return store.getters["plotSelections/selectedIndicator"];
+    });
+
+    const updateOutputColourScale = (newScaleSettings: ScaleSettings) => {
```

Comment:
I'm getting a bit confused between input/output data here. Are you going to add an `updateInputColourScale` too?

Reply:
Yeah so I am expecting this will change when we add the input back in. You can see this a bit in the next PR https://github.com/mrc-ide/hint/pull/944/files#diff-f004fdd6d2702a582a3fac1b2710c566ab7e15bcc8b53691b59229a3f76d8f25R41 but I am expecting that this will change to be `updateColourScale` and it will pull out the "active" plot from state and then commit a mutation to update the relevant scale based on this.

Reply:
OK, so the active plot with then potentially be an input one rather than an output one, depending on the current step?

Reply:
Hmm yeah either have the active plot be an input one. Or just both the "activePlot" and the step they're on to determine where to pull data from state